docs/microtvm/index.rst

```diff
@@ -0,0 +1,72 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _microtvm-index:
+
+microTVM: TVM on bare-metal
+====================================
+
+microTVM runs TVM models on bare-metal (i.e. IoT) devices. microTVM depends only on the C standard
+library, and doesn't require an operating system to execute. microTVM is currently under heavy
+development.
+
+.. figure:: https://raw.githubusercontent.com/tvmai/web-data/main/images/dev/microtvm_workflow.svg
+   :align: center
+   :width: 85%
+
+microTVM is:
+
+* an extension to TVM's compiler to allow it to target microcontrollers
+* a way to run the TVM RPC server on-device, to allow autotuning
+* a minimal C runtime that supports standalone model inference on bare metal devices.
+
+Supported Hardware
+~~~~~~~~~~~~~~~~~~~
+
+microTVM currently tests against Cortex-M microcontrollers with the Zephyr RTOS; however, it is
+flexible and portable to other processors such as RISC-V and does not require Zephyr. The current
+demos run against QEMU and the following hardware:
+
+* `STM Nucleo-F746ZG <https://www.st.com/en/evaluation-tools/nucleo-f746zg.html>`_
+* `nRF 5340 Preview Development Kit <https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF5340-PDK>`_
+
+
+Getting Started with microTVM
+~~~~~~~~~~~~~~~~~~~~~
+
+Before working with microTVM, we recommend you have a supported development board. Then, follow these
+tutorials to get started with microTVM:
+
+1. :doc:`Start the microTVM Reference VM </tutorials/micro/micro_reference_vm>`. The microTVM tutorials
```

Comment:
consider use label and reference. 
https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page

Reply:
I tried this but it didn't seem to work--unfortunately now it's been a few weeks and I can't recall why. I think because I was linking to a tutorial?

Reply:
see
- https://github.com/apache/tvm/blob/main/tutorials/get_started/cross_compilation_and_rpc.py#L18
- https://raw.githubusercontent.com/apache/tvm/main/docs/deploy/index.rst (`:ref:`tutorial-cross-compilation-and-rpc`_
- https://tvm.apache.org/docs/deploy/

Reply:
fixed, not sure what I did before but this works fine